testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -3613,6 +3613,20 @@ public void testUpdateWithPredicates()
             assertUpdate("UPDATE " + tableName + " SET b = b * 2 WHERE b = 22", 1);
             assertQuery("SELECT * FROM " + tableName, "VALUES (0, 2, 3), (11, 12, 14), (21, 44, 23)");
         }
+    }
+
+    @Test
+    public void testUpdateRowType()
```

Comment:
This needs to be overridden in `BaseHiveConnectorTest`

Reply:
Done.